pokecat/__init__.py

```diff
@@ -217,15 +217,20 @@ def populate_pokeset(pokeset, skip_ev_check=False):
 
     # check gender
     gender = pokeset["gender"]
+    expected_gender = pokeset["species"]["gender"]
+    if gender is None:
+        gender = expected_gender
```

Comment:
This looks like a change in functionality. Previously, setting the gender to null meant "no gender", not it would mean "use default gender" with no way of explicitly giving a pokemon no gender.

Reply:
This is true, however giving no gender to a Pokemon that should have one is already discouraged because it can crash PBR. If the Pokemon is correctly given no gender, then after defaulting the gender will still be null. This shouldn't functionally change any existing sets (and I will double-check to make sure it doesn't).

Reply:
Alright, that works